lesson26/script.js

```diff
@@ -0,0 +1,107 @@
+// 1.Создать объект, описывающий автомобиль (производитель, модель, год выпуска, средняя скорость), и следующие функции для работы с этим объектом:
+// Функция для вывода на экран информации об автомобиле;
+// Функция для подсчета необходимого времени для преодоления переданного расстояния со средней скоростью. Учтите, что через каждые 4 часа дороги водителю необходимо делать перерыв на 1 час.
+
+const myCar = {
+  manuf: "Subaru",
+  model: "Legacy",
+  year: "2001",
+  speed: 100,
+};
+function allData() {
+  myCar.manuf, myCar.model, myCar.year, myCar.speed;
+}
+function time(s) {
+  let t = s / myCar.speed;
+  if (t > 4) {
+    let relax = parseInt(t / 4);
+    t = t + relax;
+  }
+  alert(t);
+}
+
+//2.Создать объект, хранящий в себе отдельно числитель и знаменатель дроби, и следующие функции для работы с этим объектом:
+
+// Функция сложения 2-х объектов-дробей;
+
+// Функция вычитания 2-х объектов-дробей;
+
+// Функция умножения 2-х объектов-дробей;
+
+// Функция деления 2-х объектов-дробей;
+
+// Функция сокращения объекта-дроби.
+
+const fraction = {
+  numerator: 10,
+  denominator: 20,
+};
+const fraction2 = {
+  numerator2: 30,
+  denominator2: 40,
```

Comment:
```suggestion
  numerator: 30,
  denominator: 40,
```
в этом нет нужды, они в разных обьектах